installers/binstall/scripts/windows/install.ps1

```diff
@@ -37,7 +37,11 @@ function Install-Binary($rover_install_args) {
 }
 
 function Download($version) {
-  $url = "https://github.com/apollographql/rover/releases/download/$version/rover-$version-x86_64-pc-windows-msvc.tar.gz"
+  $base_url = $env:APOLLO_ROVER_BINARY_REMOTE
```

Comment:
It looks like this should this be renamed to match the ENV var in the nix/install.sh file as well.

Reply:
good catch. I thought i had that, but it must have gotten lost during a rebase.